specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/definitions.json

```diff
@@ -1939,6 +1939,26 @@
       ],
       "description": "Authorization header information."
     },
+    "BackendManagedIdentityCredentials": {
+      "type": "object",
+      "properties": {
+        "resource": {
+          "type": "string",
+          "description": "The application ID of the target web API (secured resource) in Microsoft Entra ID.",
```

Comment:
>       "type": "string", [](http://example.com/codeflow?start=3&length=24)

if this has a format of a uuid pl specify that. pl check other properties as well